modules/core/src/shaderlib/index.js

```diff
@@ -19,17 +19,17 @@
 // THE SOFTWARE.
 
 import {registerShaderModules, setDefaultShaderModules} from 'luma.gl';
-import {fp32, fp64, picking, phonglighting as lighting} from 'luma.gl';
+import {fp32, fp64, picking, phonglighting} from 'luma.gl';
```

Comment:
Other modules use camelCase: `phongLighting` for identifiers matching hyphenated words. Please follow unless good reason not to.